src/ResourceManager/ApiManagement/Commands.ApiManagement/AzureRM.ApiManagement.psd1

```diff
@@ -78,41 +78,43 @@ FunctionsToExport = @()
 
 # Cmdlets to export from this module, for best performance, do not use wildcards and do not delete the entry, use an empty array if there are no cmdlets to export.
 CmdletsToExport = 'Add-AzureRmApiManagementRegion', 
-               'Get-AzureRmApiManagementSsoToken', 
```

Comment:
@solankisamir please make sure these changes are reflected in `Az.ApiManagement.psd1`

Reply:
done